app/src/main/java/com/example/drawingApp/CustomViews/DrawingFieldView.kt

```diff
@@ -37,10 +40,30 @@ class DrawingFieldView @JvmOverloads constructor(
     private var path = Path()
     private var drawingCanvas: Canvas? = null
 
+    /**
+     * used for setting our previous bitmap from rotation, no expectation of having drawn
+     */
     fun setBitmap(bitmap: Bitmap) {
         canvasBitmap = bitmap
     }
 
+    /**
+     * sets the bitmap from the image obtained from the gallery.
+     */
+    fun getBitmapFromUriString(uri: Uri, context: Context) {
```

Comment:
The ticket talked about making a method to get a bitmap from file, not about getting it in one specific View. Make this a util function in its own object file and have it return a Bitmap

Reply:
okily dokily